.drone.yml

```diff
@@ -904,6 +904,150 @@ steps:
+      - chmod -R u+rw $WORKSPACE_DIR
+      - rm -rf $WORKSPACE_DIR/go $WORKSPACE_DIR/.ssh
+
+  - name: Check out code
+    environment:
+      GITHUB_PRIVATE_KEY:
+        from_secret: GITHUB_PRIVATE_KEY
+      WORKSPACE_DIR: /tmp/push-build-darwin-amd64-fips
+      TELEPORT_DIR: /tmp/push-build-darwin-amd64-fips/go/src/github.com/gravitational/teleport
+      GODL_DIR: /tmp/push-build-darwin-amd64-fips/go/src/github.com/gravitational/godl
+      BORINGCRYPTO_BRANCH: dev.boringcrypto.go1.15
+    commands:
+      - set -u
+      - mkdir -p $TELEPORT_DIR $GODL_DIR
+      - cd $GODL_DIR
+      - git clone https://github.com/gravitational/godl.git .
+      - git checkout $BORINGCRYPTO_BRANCH
+      - cd $TELEPORT_DIR
+      - git clone https://github.com/gravitational/teleport.git .
+      - git checkout ${DRONE_TAG:-$DRONE_COMMIT}
+      # fetch enterprise submodules
+      # suppressing the newline on the end of the private key makes git operations fail on MacOS
+      # with an error like 'Load key "/path/.ssh/id_rsa": invalid format'
+      - mkdir -m 0700 $WORKSPACE_DIR/.ssh && echo "$GITHUB_PRIVATE_KEY" > $WORKSPACE_DIR/.ssh/id_rsa && chmod 600 $WORKSPACE_DIR/.ssh/id_rsa
+      - ssh-keyscan -H github.com > $WORKSPACE_DIR/.ssh/known_hosts 2>/dev/null
+      - chmod 600 $WORKSPACE_DIR/.ssh/known_hosts
+      - GIT_SSH_COMMAND='ssh -i $WORKSPACE_DIR/.ssh/id_rsa -o UserKnownHostsFile=$WORKSPACE_DIR/.ssh/known_hosts -F /dev/null' git submodule update --init e
+      # this is allowed to fail because pre-4.3 Teleport versions don't use the webassets submodule
+      - GIT_SSH_COMMAND='ssh -i $WORKSPACE_DIR/.ssh/id_rsa -o UserKnownHostsFile=$WORKSPACE_DIR/.ssh/known_hosts -F /dev/null' git submodule update --init --recursive webassets || true
+      # set version
+      - if [[ "${DRONE_TAG}" != "" ]]; then echo "${DRONE_TAG##v}" > $WORKSPACE_DIR/go/.version.txt; else make print-version > $WORKSPACE_DIR/go/.version.txt; fi; cat $WORKSPACE_DIR/go/.version.txt
+      - echo "$BORINGCRYPTO_BRANCH" > $WORKSPACE_DIR/go/.bc_commit.txt
+      - cat $WORKSPACE_DIR/go/.bc_commit.txt
+      - rm -rf $WORKSPACE_DIR/.ssh
+      - mkdir -p $WORKSPACE_DIR/go/cache
+
+  - name: Build the boringcrypto Go builder
+    environment:
+      WORKSPACE_DIR: /tmp/push-build-darwin-amd64-fips
+      GODL_DIR: /tmp/push-build-darwin-amd64-fips/go/src/github.com/gravitational/godl
+    commands:
+      - set -exu
+      - cd $GODL_DIR
+      - mkdir build
+      - export BORINGCRYPTO_BRANCH=$(cat $WORKSPACE_DIR/go/.bc_commit.txt)
+      - go build -o $GODL_DIR/build/$BORINGCRYPTO_BRANCH $BORINGCRYPTO_BRANCH/main.go
+
+  - name: Pull boringcrypto build tarball from S3
+    environment:
+      AWS_S3_BUCKET:
+        from_secret: AWS_S3_BUCKET
+      AWS_ACCESS_KEY_ID:
+        from_secret: AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: AWS_SECRET_ACCESS_KEY
+      AWS_REGION: us-west-2
+      BORINGCRYPTO_BRANCH: dev.boringcrypto.go1.15
```

Comment:
Should be set using `export BORINGCRYPTO_BRANCH=$(cat $WORKSPACE_DIR/go/.bc_commit.txt)` in commands as above and removed here.